app/presenters/tree_node/menu/item.rb

```diff
@@ -0,0 +1,39 @@
+module TreeNode
+  module Menu
+    class Item < TreeNode::Menu::Node
+      set_attribute(:key)     { "#{@options[:node_id_prefix]}__#{@object.feature}" }
+      set_attribute(:title)   { _(details[:name]) }
+      set_attribute(:tooltip) { _(details[:description]) || _(details[:name]) }
+
+      def select_state
+        self_selected? || select_by_children_selected
+      end
+
+      def children
```

Comment:
I would like to have something like this in the future, but for now let's put this into the `TreeBuilder` to be on the same abstraction level with the other `TreeNode`s. As we have all trees generated in the same way, we can go further with this...